### PR TITLE
[WIP] refactor: use base queries to inherit

### PR DESCRIPTION
**Pull request by elianiva**
I made a separate kind for each braces so we can easily combine them when inheriting. Let me know what you think.

### CODE REVIEW

**Review by theHamsta — APPROVED** [review]
Nice!

File: queries/quote/pairs.scm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+(_
+  "\"" @left
+  "\"" @right)
```
Comment (theHamsta) [inline queries/quote/pairs.scm:3]: Maybe have this also for single quote?

**Comment by elianiva** [comment]
okay, I think I'm done.

**Comment by theHamsta** [comment]
Thanks a lot for the work!

**Pull request merged**